src/Generator/ServiceProviderGenerator.php

```diff
@@ -221,7 +232,7 @@ private function generateBodySerializerClosure(Specification $specification): Cl
 
         $registerBodySerializerClosureStatements[] = $this
             ->builder
-            ->return($xmlSerializerInit ?? $formEncodedSerializerInit ?? $jsonSerializerInit ?? $initialStatement);
+            ->return($xmlSerializerInit ?? $formEncodedSerializerInit ?? $jsonSerializerInit ?? $vdnApiJsonContentTypeSerializer ?? $initialStatement);
```

Comment:
and then
```suggestion
            ->return($vdnApiJsonContentTypeSerializer ?? $xmlSerializerInit ?? $formEncodedSerializerInit ?? $jsonSerializerInit ?? $initialStatement);
```
otherwise you need to change the methodCall for FormEncoded and Xml as well